.gitignore

```diff
@@ -27,6 +27,3 @@ build
 # JetBrains
 .idea/
 .ignore/
-
-admin/social-income-prod-firebase-adminsdk.json
```

Comment:
Shall we keep this, so that no one accidentally commits this? I think it's currently used to run the scripts, right?

Reply:
Yes, let's keep it👍🏼

In the future, the scripts should also run against the local firestore imo. We could integrate some buttons into the admin UI to trigger them in production, what do you think?